### DISCUSSION TITLE
Can I click the node in the DAG diagram?

### DISCUSSION THREAD

**Discussion by liujie316316** [apache/airflow#29829] [Q&A]
I want to put the DAG diagram generated by airflow into QT, and the nodes in the DAG diagram can be clicked. What should I do? I can't click the DAG graph node in the airwindow ui!!!

**Comment by potiuk** [comment]
You can export DAG with Airflow CLI to a ".dot" format with `airflow dags show` command: https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#show and do whatever you want with the .dot format (post-process it to add URLs., convert it to whatever format you want - there are many tools that work with .dot. Also you can use `airflow dags export` to export DAG to json for example and render it in whatever form you find useful.
